app/controllers/features_controller.rb

```diff
@@ -88,4 +90,15 @@ def update
       end
     end
   end
+
+  def sort_feature(features, sorting_type)
+    case sorting_type
+    when 'title'
+      features.sort_by(&:title.downcase)
+    when 'last_edited'
+      features.sort_by(&:changed).reverse!
+    else
+      features
+    end
+  end
```

Comment:
Thanks for working on this...lets move this to the model...actually we had another method for sorting in the node model...we can reuse that...Let me see if we have merged that PR